infogami/core/db.py

```diff
@@ -4,13 +4,18 @@
 import infogami
 from infogami.utils.view import public
 
+try:
+    basestring
+except NameError:
+    basestring = (str, )
+
 def get_version(path, revision=None):
     return web.ctx.site.get(path, revision)
 
 @public
 def get_type(path):
     return get_version(path)
-    
+
```

Comment:
All these whitespace changes make the diff very noisy

Reply:
Sorry but today most editors and conversion tools (e.g. futurize, modernize) will automatically whack trailing whitespace in .py files.  It is a bit noisy now but should save aggravation on future PRs.